ircspeak.sh

```diff
@@ -1,2 +1,2 @@
 #! /bin/sh
-shuf ~/irclogs/freenode/\##zuzakistan.log -n 1
+shuf -n 1 ~/irclogs/freenode/\##zuzakistan.log | sed -r 's/^[0-9-]+T[0-9:+].*-[A-Za-z0-9]*:#*[A-Za-z0-9]*- (.*)/\1/'
```

Comment:
It might be better to do this in the Javascript part instead of the Shell part; eventually we'll want to remove the shell stuff entirely